packages/@aws-cdk/core/lib/asset-staging.ts

```diff
@@ -137,15 +142,31 @@ export class AssetStaging extends Construct {
     }
   }
 
-  private bundle(options: BundlingOptions): string {
+  private bundle(options: BundlingOptions, assetHash?: string): string {
```

Comment:
add a docstring with "@property" explaining why `assetHash` is passed here and what happens if it does not

Reply:
Rename to `sourceHash`

Reply:
Done.